pulsar/producer_partition.go

```diff
@@ -126,6 +126,9 @@ func newPartitionProducer(client *client, topic string, options *ProducerOptions
 		metrics:          metrics,
 		epoch:            0,
 	}
+	if p.options.DisableBatching {
+		p.batchFlushTicker.Stop()
```

Comment:
Is it ok to call Stop() multiple times? There is a call to stop in the internal close.

Reply:
It's ok